wagtail/project_template/home/templates/home/welcome_page.html

```diff
@@ -0,0 +1,380 @@
+                <path fill="#231F20" d="M127,83.9h-8.8l-12.6-36.4h7.9l9,27.5l9-27.5h7.9l9,27.5l9-27.5h7.9L153,83.9h-8.8L135.6,59L127,83.9z"/>
+                <path fill="#231F20" d="M200.1,83.9h-7V79c-3,3.6-7,5.4-12.1,5.4c-3.8,0-6.9-1.1-9.4-3.2c-2.5-2.1-3.7-5-3.7-8.6
+                  c0-3.6,1.3-6.3,4-8c2.6-1.8,6.2-2.7,10.7-2.7h9.9v-1.4c0-4.8-2.7-7.3-8.1-7.3c-3.4,0-6.9,1.2-10.5,3.7l-3.4-4.8
+                  c4.4-3.5,9.4-5.3,15.1-5.3c4.3,0,7.8,1.1,10.5,3.2c2.7,2.2,4.1,5.6,4.1,10.2V83.9z M192.4,70.3v-3.1h-8.6c-5.5,0-8.3,1.7-8.3,5.2
+                  c0,1.8,0.7,3.1,2.1,4.1c1.4,0.9,3.3,1.4,5.7,1.4c2.4,0,4.6-0.7,6.4-2.1C191.5,74.5,192.4,72.7,192.4,70.3z"/>
+                <path fill="#231F20" d="M241.7,47.5v31.7c0,6.4-1.7,11.3-5.2,14.5c-3.5,3.2-8,4.8-13.4,4.8c-5.5,0-10.4-1.7-14.8-5.1l3.6-5.8
+                  c3.6,2.7,7.1,4,10.8,4c3.6,0,6.5-0.9,8.6-2.8c2.1-1.9,3.2-4.9,3.2-9v-4.7c-1.1,2.1-2.8,3.9-4.9,5.1c-2.1,1.3-4.5,1.9-7.1,1.9
+                  c-4.8,0-8.8-1.7-11.9-5.1c-3.1-3.4-4.7-7.6-4.7-12.6c0-5,1.6-9.2,4.7-12.6c3.1-3.4,7.1-5.1,11.9-5.1c4.8,0,8.7,2,11.7,6v-5.4
+                  H241.7z M213.3,64.3c0,3,0.9,5.6,2.8,7.7c1.8,2.2,4.3,3.2,7.5,3.2c3.1,0,5.7-1,7.6-3.1c1.9-2.1,2.9-4.7,2.9-7.8
+                  c0-3.1-1-5.8-2.9-7.9c-2-2.2-4.5-3.2-7.6-3.2c-3.1,0-5.6,1.1-7.4,3.4C214.2,58.7,213.3,61.3,213.3,64.3z"/>
+                <path fill="#231F20" d="M260.9,53.6v18.5c0,1.7,0.5,3.1,1.4,4.1c0.9,1,2.2,1.5,3.8,1.5c1.6,0,3.2-0.8,4.7-2.4l3.1,5.4
+                  c-2.7,2.4-5.7,3.6-8.9,3.6c-3.3,0-6-1.1-8.3-3.4c-2.3-2.3-3.5-5.3-3.5-9.1V53.6h-4.6v-6.2h4.6V36.1h7.7v11.4h9.6v6.2H260.9z"/>
+                <path fill="#231F20" d="M309.5,83.9h-7V79c-3,3.6-7,5.4-12.1,5.4c-3.8,0-6.9-1.1-9.4-3.2c-2.5-2.1-3.7-5-3.7-8.6
+                  c0-3.6,1.3-6.3,4-8c2.6-1.8,6.2-2.7,10.7-2.7h9.9v-1.4c0-4.8-2.7-7.3-8.1-7.3c-3.4,0-6.9,1.2-10.5,3.7l-3.4-4.8
+                  c4.4-3.5,9.4-5.3,15.1-5.3c4.3,0,7.8,1.1,10.5,3.2c2.7,2.2,4.1,5.6,4.1,10.2V83.9z M301.8,70.3v-3.1h-8.6c-5.5,0-8.3,1.7-8.3,5.2
+                  c0,1.8,0.7,3.1,2.1,4.1c1.4,0.9,3.3,1.4,5.7,1.4c2.4,0,4.6-0.7,6.4-2.1C300.9,74.5,301.8,72.7,301.8,70.3z"/>
+                <path fill="#231F20" d="M319.3,40.2c-1-1-1.4-2.1-1.4-3.4c0-1.3,0.5-2.5,1.4-3.4c1-1,2.1-1.4,3.4-1.4c1.3,0,2.5,0.5,3.4,1.4
+                  c1,1,1.4,2.1,1.4,3.4c0,1.3-0.5,2.5-1.4,3.4s-2.1,1.4-3.4,1.4C321.4,41.7,320.3,41.2,319.3,40.2z M326.5,83.9h-7.7V47.5h7.7V83.9z
+                  "/>
+                <path fill="#231F20" d="M342.5,83.9h-7.7V33.1h7.7V83.9z"/>
+              </g>
+            </g>
+          </svg>
+        </a>
+      </div class="logo">
+    </div class="column">
+    <div class="column">
+      {% templatetag openblock %} blocktrans {% templatetag closeblock %}View the <a href="http://docs.wagtail.io/en/latest/releases/" target="_blank" rel="noopener">release notes </a>{% templatetag openblock %} endblocktrans {% templatetag closeblock %}
+    </div class="column">
+  </div class="row">
+</header>
+<main>
```

Comment:
IE11 does not support the `main` tag, please use `div role="main"` instead.

Reply:
Fixed.